README.md

```diff
@@ -175,10 +175,26 @@ non-deterministic.
 Output files are text files that contain one record per line (i.e.,
 they're separated by `\n`).
 
+There are two types of data format available: 
+ - **[Default]** Flat structure, where field values are separated by comma (`csv`)
+    
+    Configuration: ```format.output.type=csv```. 
+    Also, this is the default if the property is not present in the configuration.
+    
+ - Complex structure, where fields and values represented as a JSON object (`jsonl`)
```

Comment:
```suggestion
 - Complex structure, where fields and values represented as a [JSON lines](https://jsonlines.org/) object (`jsonl`)
```

Reply:
I think that here we are talking about lines, not a file itself.
In every line structure is JSON object. 
If this is not clear, I could add something like: 


Output files are text files that contain one record per line (i.e.,
they're separated by `\n`).

There are two types of data format available **to write each record**:

Reply:
- [ ] Reformulate description